vmaas/load.go

```diff
@@ -375,10 +375,11 @@ func loadRepoDetails(c *Cache) { //nolint: funlen
 	prodID2RepoIDs := make(map[int][]RepoID, cntProd)
 	repoIDs := []RepoID{}
 	var repoID RepoID
+	var lastChange *string
```

Comment:
since it is never NULL
```suggestion
	var lastChange string
```